node/helpers_test.go

```diff
@@ -47,15 +45,12 @@ func TestGetNodeHeight(t *testing.T) {
 	bmConfig := getBMConfig()
 	fullNode, _ := createAndConfigureNode(ctx, 0, true, false, keys, bmConfig, dalc, t)
 	lightNode, _ := createNode(ctx, 1, true, true, keys, bmConfig, t)
-	require.NoError(fullNode.Start())
-	defer func() {
-		assert.NoError(fullNode.Stop())
-	}()
 
-	assert.NoError(lightNode.Start())
-	defer func() {
-		assert.NoError(lightNode.Stop())
-	}()
+	fullNode.(*FullNode).dalc = dalc
+	fullNode.(*FullNode).blockManager.SetDALC(dalc)
```

Comment:
Why those lines are introduced? `dalc` is already passed to `createAndConfigureNode`?